docs/src/main/asciidoc/management-interface-reference.adoc

```diff
@@ -0,0 +1,180 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Management interface reference
+include::_attributes.adoc[]
+:categories: observability
+:summary: Management interface configuration
+:numbered:
+:sectnums:
+:sectnumlevels: 4
+
+By default, Quarkus exposes the _management_ endpoints under `/q` on the main HTTP server.
+The same HTTP server provides the application endpoints and the management endpoints.
+
+This document presents how you can use a separate HTTP server (bound to a different network interface and port) for the management endpoints.
+It avoids exposing these endpoints on the main server and, therefore, prevents undesired accesses.
+
+== Enabling the management interface
+
+To enable the management interface, use the following **build-time** property:
+
+[source, properties]
+----
+quarkus.management.enabled=true
```

Comment:
@cescoffier for clarification — without this option, everything works as it now, and served on `localhost:8080/q` (including, say, arc and Dev UI); and without this option it is served on `${quarkus.management.host}:${q.m.port}${q.m.root-path}`, by default `localhost:9000/q` ? In that case, `quarkus.management.enabled` is probably not the best name, since(from user's perspective), we do not enable something but merely moving it to a separate address.

Reply:
What property name would you propose?

For the framework it enables the second server.

Reply:
Maybe something like `management.server.enabled`? Or `management.separate-server`?

Reply:
Yes, we don't expose all /q, just the management endpoints.